src/main/java/com/eppo/sdk/helpers/VariationHelper.java

```diff
@@ -23,6 +23,6 @@ static public Variation selectVariation(String inputKey, int subjectShards, List
     }
 
     static public double variationProbability(Variation variation, int subjectShards) {
-        return (double)(variation.getShardRange().end - variation.getShardRange().start + 1) / subjectShards;
```

Comment:
This math was wrong because the end of the range is _exclusive_.

This was caught by the unit test finding 0.3335 probability -- yay unit tests!